### ISSUE TITLE
Wrong symbol over USB

### ISSUE THREAD

**Issue by PaPa1ya** [tejado/Authorizer#51]
If the password contains "-", then when entering it via usb, from the page with all passwords, this symbol is replaced by "/". If you go to the page with the password and press "PASSWORD", the input will be correct.

**Comment by tejado** [comment]
Yes, they keyboard language is there hardcoded and the system setting is ignored: https://github.com/tejado/Authorizer/blob/master/authorizer/src/main/java/net/tjado/passwdsafe/PasswdSafe.java#L1219 - this needs to be fixed.
Pull Requests are highly welcome!
